Objects/codeobject.c

```diff
@@ -12,6 +12,62 @@
 #include "clinic/codeobject.c.h"
 
 
+static void
+notify_code_watchers(PyCodeEvent event, PyCodeObject *co)
```

Comment:
Could you add a (8) bit vector of watchers, much as we did for function watchers.
That way, only one test is needed in the common case where there are no watchers.

Reply:
sure, let me add it